client/src/modules/about.module.css

```diff
@@ -0,0 +1,46 @@
+@import url('https://fonts.googleapis.com/css2?family=Abril+Fatface&family=Baloo+2:wght@400;600;800&family=PT+Serif+Caption&display=swap');
+
+.root {
+    display: inline;
+    height: 100%;
+}
+
+.container {
+    display: flex;
+    width: 926px;
+    height: 75px;
+    margin: auto;
+}
+
+.img {
+    border-radius: 50%;
+    object-fit: cover;
+    height: 230px;
+    width: 230px;
+}
+
+.square {
+    border-radius: 7.5px;
+    display: flex;
+    flex-direction: column-reverse;
+    height: 380px;
+    margin: 30px auto;
+    padding: 10px 10px 50px;
+    text-align: left;
+    width: 380px;
+}
+
+@media (max-width: 768px) {
+    .container {
+        display: block;

```

Comment:
width of the `container` class should be adjusted as well

Reply:
done!